common/config/config.go

```diff
@@ -652,3 +678,78 @@ func (p *JWTKeyProvider) HasSourceURIsConfigured() bool {
 	}
 	return false
 }
+
+func (rpc *RPC) GetKeepAliveServerParameters() keepalive.ServerParameters {
+	// 0 are treated as default values in grpc: https://github.com/grpc/grpc-go/blob/43a4a84abcd406230e7471062dd2ee5c7ba2485d/internal/transport/http2_server.go#L215-L225
+	// the default config is same as grpc default config, same for the below client config and enforcement policy
+	defaultConfig := keepalive.ServerParameters{
+		MaxConnectionIdle:     0 * time.Minute,
+		MaxConnectionAge:      0,
+		MaxConnectionAgeGrace: 0 * time.Second,
+		Time:                  2 * time.Hour,
+		Timeout:               20 * time.Second,
+	}
+	if rpc.KeepAliveServerConfig == nil {
```

Comment:
```suggestion
	if rpc == nil || rpc.KeepAliveServerConfig == nil {
```
rpc == nil shouldn't happen, but maybe it makes sense to be defensive and not panic?